### DISCUSSION TITLE
x10 with Heyu Unable to send address bytes

### DISCUSSION THREAD

**Discussion by tommyd75** [HeyuX10Automation/heyu#58] [Q&A]
I keep getting this error when I try to manually send an x10 on command to K9, has anyone ever seen this error?
"Unable to send address bytes"

I have been trying it with a IBM Homedirector HD11A and a RCA HC60CRX versions of the CM11 but they gave the same error, but just tried with a XTB232 and it works fine. Must be something specific to those other devices. Has anyone ever seen this?

See below for what happens when I run heyu info

pi@raspberrypi:~ $ heyu info
Heyu version 2.10
Configuration at /etc/heyu/x10.conf
Powerline interface on /dev/ttyS0
Firmware revision Level = 1
Interface battery usage = Unknown
Raw interface clock: Wed, Day 290, 19:42:10
(--> Civil Time: Wed 18 Oct 2017 20:42:10 CDT)
No schedule has been uploaded by Heyu.
Housecode = K
0 = off, 1 = on, unit 16.......8...4..1
Last addressed device = 0x0000 (0000000000000000)
Status of monitored devices = 0x0000 (0000000000000000)
Status of dimmed devices = 0x0000 (0000000000000000)
pi@raspberrypi:~ $ heyu on k9
Unable to send address bytes

**Comment by chrismillsuk** [comment]
I get this error, I found that when  I run heyu info once it returns my correct house code, when I run it again a few seconds later the house code changes from B to J and any subsequent commands to heyu ie heyu on B5 fail. 

Stopping heyu and modprobe -r the driver and reloading it seems to reset it then it fails again....

**Comment by jkrzyszt** [comment]
Hardware issue, I guess.  Converting to a QA discussion, with no answer though.
